build.gradle

```diff
@@ -47,6 +47,14 @@ subprojects {
 
 apply from: 'cppSettings.gradle'
 
+def formatArgs = project.hasProperty('formatArgs') ? project.formatArgs : ''
+
+task format(type: Exec) {
+    workingDir "${rootDir}"
+    commandLine 'python'
+    args 'styleguide/format.py', formatArgs
```

Comment:
That's not how we want to do this, I think. I'm pretty sure this won't handle multiple arguments correctly.